### DISCUSSION TITLE
Let's deprecate broad region tags

### DISCUSSION THREAD

**Discussion by Dum4G** [iptv-org/community#281] [Ideas]
When people download their country playlists they definitely expect to see channels that are broadcasted within their region in a language they can understand. **I think we should only preserve tags that imply simillarity in languages and culture (like CIS, HISPAM) and avoid use of way too broad and sort of abusive tvg-country tags like EMEA, Europe, Asia etc.** For very special cases (like Al jazeera, BBC, DW) the International tag can be used instead. A good examples to prove importance of that change are https://github.com/iptv-org/iptv/blob/gh-pages/countries/by.m3u and https://github.com/iptv-org/iptv/blob/gh-pages/countries/ru.m3u that suddenly have an Australian channel and AXN East Asia in them. In fact Belarus playlist contains more channels with inappropriate tags than local ones.

**Comment by Dum4G** [comment]
@freearhey @LaneSh4d0w @Sphinxroot @sguinetti @MapGuy11 @dct-infra

**Reply by BellezaEmporium** [reply]
The thing is those are official notations, changing them would be nonsense whatsoever. If there's some channels that do not seem to be correctly in place, we need to retarget them.

**Reply by Dum4G** [reply]
Despite that, do you really think there's a channel that can be interesting for entire Europe or entire Asia? Especially if it's not an English stream? Rather than changing those notations I'm suggesting to avoid using them

**Reply by BellezaEmporium** [reply]
**interesting** probably not, but **targeted to** yes.

There's channels that are internationally casted, depending on where you are in the continent. There might be variants, but overall there's well known channels that were exported worldwide.

As an example, with my french ISP, I have certain access to Chinese channels (via paid pack : Phoenix Television, CCTV, Dragon Television etc...)

**Reply by sguinetti** [reply]
As I share the opinion of @LaneSh4d0w, it is weird that some channels allow to cover the signal of other countries where they plan to transmit via extension packs. In Peru, there are cable packages to watch international TV such as CCTV 4 and NHK. So it would be good to document how channels with international coverage works (like language, program, etc).

**Reply by loucasal** [reply]
A wider problem (at least for playlists by country) is that the same channels get duplicated in multiple playlists, or even create country playlists that wouldn't exist otherwise. I think many users, given the option, would prefer to place them in a broader (e.g. regional or international) grouping rather than having so many duplicates. Part of what I was proposing in iptv-org/iptv-org.github.io#29.

**Reply by BellezaEmporium** [reply]
The regional playlist was made for this occasion, see repository README.

**Reply by loucasal** [reply]
The regional playlist doesn't solve the issue: some channels are local by definition, others are national, others are regional/international. A one size fits all approach will never work for all channels. See also the linked discussion.

**Comment by freearhey** [comment]
If a channel should not be in the playlist of some country, it is enough to fix `broadcast_area` of this channel in the [iptv-org/database](https://github.com/iptv-org/database) and at the next update it will disappear from it.

https://github.com/iptv-org/database/blob/master/CONTRIBUTING.md#how-to-edit-feed-description
